### DISCUSSION TITLE
Bedwars Loot Boxes Stats

### DISCUSSION THREAD

**Discussion by Polsulpicien** [HypixelDev/PublicAPI#513] [Q&A] [answered]
Hello! I wanted to know if you could tell me the difference between:
`player/achievements/bedwars_loot_boxes` (What is this ?)
`player/stats/Bedwars/bedwars_boxes` (number of boxes available I think ?)
`player/stats/Bedwars_openedChests` (What is this ?)

Chests types, but when you add them, you don't get the same number than `Bedwars_openedChests` 🤔 
`player/stats/Bedwars_openedCommons` 
`player/stats/Bedwars_openedRares`
`player/stats/Bedwars_openedEpics`
`player/stats/Bedwars_openedLegendaries`
Thank you :)

**Comment by Polsulpicien** [comment]
Anyone knows ? :/

**Comment by Picsou993** [comment, marked as answer]
Hi.
`player.achievements.bedwars_loot_boxes` is the amount of loot boxes opened by the player. This is the tracking used for the tiered achievement **Crate Looter**. It counts all boxes no matter of it's a special box (golden loot box, etc...).

`player.stats.Bedwars.bedwars_boxes` is the amount of Normal Loot Boxes that the player has. You can find the special loot boxes using `bedwars_hallowe_boxes`, `bedwars_golden_boxes`, etc...

`player.stats.Bedwars.Bedwars_openedChests` is the amount of rewards from loot boxes. When you open a loot box, you have 3 rewards.


For `Bedwars_openedCommons`, `Bedwars_openedRares`, `Bedwars_openedEpics` and `Bedwars_openedLegendaries`, they will usually sum to `Bedwars_openedChests` but in some cases not. This is the case for a lot of statistics that should sum or be the same.

**Comment by Polsulpicien** [comment]
Thank you very much for your help, appreciate it 😄
